README.md

```
@@ -346,3 +346,21 @@ kind create cluster --name easyshop
 # Deploy the application locally
 kubectl apply -f kubernetes/
 ```
+# test
+# test
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)